core/src/core/temporal/scheduled_workflow.py

```
@@ -0,0 +1,92 @@
+) -> None:
+    """Обновляет метаданные выполнения для запуска по расписанию."""
+    if not hasattr(pipeline_config, "execution_metadata"):
+        pipeline_config.execution_metadata = {}
+
+    pipeline_config.execution_metadata.update(
+        {
+            "scheduled": True,
+            "schedule_id": schedule_id,
+            "scheduled_time": scheduled_time,
+            "execution_type": "scheduled",
+        }
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)